README.md

```diff
@@ -20,7 +20,8 @@ This module adds a new [scheduled_task](https://puppet.com/docs/puppet/latest/ty
 
 ### Beginning with scheduled_task
 
-The scheduled_task module adapts the Puppet [scheduled_task](https://puppet.com/docs/puppet/latest/types/scheduled_task.html) resource to run using a modern API. To get started, install the module and declare 'taskscheduler_api2' as the provider, for example:
+The scheduled_task module adapts the Puppet [scheduled_task](https://puppet.com/docs/puppet/latest/types/scheduled_task.html) resource to run using a modern API.
+To get started, install the module and declare 'taskscheduler_api2' as the provider, for example:
```

Comment:
@Iristyle, @glennsarti:

I wasn't quite sure how to sensibly rewrite this section, but now that the type and provider are included in the module, installing the module clobbers the type/provider in core Puppet, right? And it replaces the default provider with the v2 API provider.

So by default, all you need to do is install the module to get the new provider - if you need to stick with the old provider, you'll need to specify it in your manifests going forward, right?

Reply:
"you'll need to specify it in your manifests going forward, right?"  Correct

Reply:
Also...bit further up;
```
provider capable of using the more modern Version 2 Windows API for task management
```
Do we need to explain why V2 is better etc?